catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.xml

```diff
@@ -222,6 +222,20 @@
             type="Boolean"
             default="false"/>
 
+        <AD id="customTextNotationEnabled"
+          name="Show Custom Text Notations"
+          description="Toggle the display of custom text notations next to metacard map markers."
+          type="Boolean"
+          default="false"
+          required="false"/>
+
+        <AD id="customTextNotationAttribute"
```

Comment:
Could we make it so that "title" is in the textbox, so we know it's the default?